packages/now-cli/src/util/validate-paths.ts

```diff
@@ -9,6 +9,33 @@ import toHumanPath from './humanize-path';
 
 const stat = promisify(lstatRaw);
 
+/**
+ * A helper function to validate the `rootDirectory` input.
+ */
+export async function validateRootDirectory(output: Output, path: string) {
+  const pathStat = await stat(path).catch(() => null);
+
+  if (!pathStat) {
+    output.print(
+      `${chalk.red('Error!')} Could not find ${chalk.cyan(
+        `“${toHumanPath(path)}”`
+      )}\n`
```

Comment:
It might be unclear where this error is coming from. Can we explain how to fix it?

Reply:
changed the error message